openquake/risklib/asset.py

```diff
@@ -141,9 +141,11 @@ def get_units(self, loss_types):
             if lt.endswith('_ins'):
                 lt = lt[:-4]
             if lt == 'area':
-                unit = 'sqm'
+                # NOTE: there's currently no trivial way to retrieve the
+                #       actual measurement unit from data (e.g. sqm or sqft)
```

Comment:
THERE IS A TRIVIAL WAY! units are listed in the exposure. For instance
```xml
case_1g/exposure_model.xml
8:    <area type="per_asset" unit="SQM" />
10:      <costType name="structural" type="per_asset" unit="USD" />
11:      <costType name="nonstructural" type="per_asset" unit="USD" />
12:      <costType name="contents" type="per_asset" unit="USD" />
13:      <costType name="business_interruption" type="per_asset" unit="USD/month" />
```

Reply:
At present, we can run the models without that tag in the exposure XML. What happens if no units are specified in that way?